Cargo.toml

```diff
@@ -46,7 +46,7 @@ dotenvy = { version = "0.15" }
 serial_test = { version = "2.0", default-features = false, features = ["async", "file_locks"] }
 tokio = { version = "1.0", default-features = false, features = ["macros", "rt-multi-thread"] }
 tracing-subscriber = { version = "0.3", default-features = false, features = ["env-filter", "fmt", "std"] }
-twilight-gateway = { version = "0.15", default-features = false }
+twilight-gateway = { version = "0.15", default-features = false, features = ["rustls-native-roots"] }
```

Comment:
We don't require any `twilight-gateway` feature ourselves so we shouldn't specify any. Users of `redlight` can specify `twilight-gateway` features in their own `Cargo.toml` which will then propagate into all dependencies that use it.

Reply:
True, didn't notice it's under `dev-dependencies`